templates/legacy-docs.html

```diff
@@ -0,0 +1,17 @@
+{% extends "skel.html" %}
```

Comment:
With this, the pages will have just `Matrix.org` as the HTML `<title>`, right? Did extending `page.html` no work?